### DISCUSSION TITLE
msix:create failed due to empty version

### DISCUSSION THREAD

**Discussion by vadimbarda** [codemagic-ci-cd/codemagic-docs#1620] [Q&A] [answered]
Hi, community. MacOS build with publishing to Store works fine, but my Windows build start failing in reason of incorrect version for msix. I see log:
`> flutter.BAT pub run msix:create --build-windows false --version ''`
But I can't understand why Codemagic run this command with empty version. Yaml file contains correct version: `version: 1.0.2`
Please help with suggestions. Build ID: 64541bea2e754d74f09a33d8

**Comment by mohammedbabelly20** [comment]
Hey @vadimbarda 
I think you're missing the msix version under your msix configurations.

```
msix_config:
  ...
  msix_version: 1.0.0.0
```

**Reply by vadimbarda** [reply, marked as answer]
No. Codemagic don't read msix_version property from yaml file. It use codemagic Package Version from Windows Distribution config section in Workflow Editor UI. In my case this config had Environment variable reference I deleted
